pkg/rulerequest/handle.go

```diff
@@ -41,23 +41,25 @@ func PromptRun(ctx context.Context, c client.Client, dangerous bool, image strin
 		if err != nil {
 			return nil, err
 		}
-		var img *apiv1.Image
-		var tag string
-		img, tag, err = images.FindImageMatch(apiv1.ImageList{Items: il}, image)
+		var exImg *apiv1.Image
+		var exImgTag, exImgName string
```

Comment:
nit: if `ex` stands for `exact` here, I recommend just including that full word in the variable names. It was unclear to me what these were for until I saw the comment farther down.

Reply:
`existing` it is :)